justfile

```diff
@@ -224,6 +224,18 @@ get-credentials-ci:
         --name "contrast-ci" \
         --admin
 
+get-credentials-tdxbm:
+    #!/usr/bin/env bash
+    set -euo pipefail
+    tmpConfig=$(mktemp)
+    gcloud secrets versions access \
+        projects/796962942582/secrets/m50-ganondorf-kubeconf/versions/2 \
+        --out-file="$tmpConfig"
```

Comment:
Do we really need this in the `justfile`? It sounds like it should be part of Github Actions or something.

Reply:
It enables devs to quickly setup the needed creds, without setting up ssh keys and searching for kube config in other users home directory. Only requirement is to be in the tailnet (should add a command with this requirement).

Reply:
On the other hand, this is only useful to a particular group of devs - those who have access to this gcp secret. I simply wonder how much org-specific things we should put into OSS, but feel free to go ahead with it.